source/patterns/@aws-solutions-constructs/aws-lambda-sagemakerendpoint/test/integ.existingSageMakerEndpoint.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ *  Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+// Imports
+import { Stack, Duration, App } from '@aws-cdk/core';
+import { LambdaToSageMakerEndpoint, LambdaToSageMakerEndpointProps } from '../lib';
```

Comment:
Let's standardize on Sagemaker (as opposed to SageMaker). Both are used throughout the code, Sagemaker (or sagemaker) is appropriate for Pascal-case and camel-case names. For instance, even though the whole world refers to SQS, when we create Pascal or camel case class name we use Sqs or sqs.

Reply:
will fix.